components/organisms/Contributors/contributors.tsx

```diff
@@ -31,17 +31,18 @@ import ContributorTable from "../ContributorsTable/contributors-table";
 interface ContributorProps {
   repositories?: number[];
   title?: string;
+  defaultLayout?: ToggleValue;
```

Comment:
nit (naming): it's just layout.

```suggestion
  layout?: ToggleValue;
```

Reply:
`layout` is already used for the `useState`; will keep as `defaultLayout` unless you have another name I should use